apps/modernization-ui/src/apps/page-builder/page/management/edit/content/PageContent.tsx

```diff
@@ -1,14 +1,34 @@
-import { PagesTab } from 'apps/page-builder/generated';
+import { PageQuestionControllerService, PagesTab } from 'apps/page-builder/generated';
 import { Sections } from '../section/Sections';
 import { PageSideMenu } from './PageSideMenu';
 import styles from './page-content.module.scss';
+import { authorization } from 'authorization';
+import { usePageManagement } from '../../usePageManagement';
+import { useAlert } from '../../../../../../alert';
 
 type Props = {
     tab: PagesTab;
 };
 export const PageContent = ({ tab }: Props) => {
-    const handleAddQuestion = (subsection: number) => {
-        console.log('add question not yet implemented', subsection);
+    const { showAlert } = useAlert();
+    const { page } = usePageManagement();
+    const handleAddQuestion = (subsection: number, questionId: number) => {
```

Comment:
Question: Is there reason this controller call is in PageContent and not in Subsection? If not then it might be better to put the add question in the Subsection since it already has the delete/edit question.

Reply:
Moved the code inside Subsection